candle-examples/examples/mnist-training/main.rs

```diff
@@ -140,7 +140,7 @@ struct Args {
 pub fn main() -> anyhow::Result<()> {
     let args = Args::parse();
     // Load the dataset
-    let m = candle_datasets::vision::mnist::load_dir("data")?;
+    let m = candle_datasets::vision::mnist::load()?;
```

Comment:
Could we have a flag to use the old data loading optionally? That would be convenient at least to me as I'm using this example fairly often.